app/views/trends/_empty_data.html.erb

```diff
@@ -0,0 +1,7 @@
+<% title(_("Trends for %s") % trend_title(@trend)) %>
+<div class="row">
+  <div class="stats-well col-md-12">
+    <p><strong><%= _('No data for this trend.') %></strong></p>
+    <div><%= _("Is the cron job that executes %s enabled?") % "<span class='black'>foreman-rake trends:counter</span>" %></div>
```

Comment:
Missing some html_safe-ness here.  It used to work, seems to have regressed.

![screenshot from 2015-07-22 09 21 39](https://cloud.githubusercontent.com/assets/482089/8821116/1a4c85fe-3053-11e5-9112-59a67c3752a8.png)